lite2/setup.go

```diff
@@ -0,0 +1,78 @@
+package lite
+
+import (
+	"time"
+
+	"time"
```

Comment:
weird to have time here twice....

Reply:
I think that's something to do with my IDE